client_tools_test.go

```diff
@@ -0,0 +1,39 @@
+package tarantool_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/vmihailenco/msgpack/v5"
+
+	"github.com/tarantool/go-tarantool/v2"
+)
+
+func TestOperations_EncodeMsgpack(t *testing.T) {
+	ops := tarantool.NewOperations().
+		Add(1, 2).
+		Subtract(1, 2).
+		BitwiseAnd(1, 2).
+		BitwiseOr(1, 2).
+		BitwiseXor(1, 2).
+		Splice(1, 2, 3, "a").
+		Insert(1, 2).
+		Delete(1, 2).
+		Assign(1, 2)
+
+	refBuf := []byte{153, 147, 161, 43, 1, 2, 147, 161, 45, 1, 2, 147, 161, 38,
+		1, 2, 147, 161, 124, 1, 2, 147, 161, 94, 1, 2, 149, 161, 58, 1, 2, 3,
+		161, 97, 147, 161, 33, 1, 2, 147, 161, 35, 1, 2, 147, 161, 61, 1, 2}
```

Comment:
Let's comment here at least on the beginning of each operation.

Reply:
Updated.